docs/import/import_data.js

```
@@ -140,132 +135,327 @@ function stringifyGeoid(feature) {
+    }
+    const buildingPath = buildingPaths[state];
+    if (!buildingPath) {
+      return missingAssetError('building asset path for ' + state);
+    }
+    const blockGroupPath = blockGroupPaths[state];
+    if (!blockGroupPath) {
+      return missingAssetError(
+          'Census TIGER block group shapefile for ' + state);
+    }
+
+    const stateGroups =
+        ee.FeatureCollection(blockGroupPath).filterBounds(damageEnvelope);
+
+    let processing = ee.FeatureCollection(snapPath).map(stringifyGeoid);
+
+    // Join snap stats to block group geometries.
+    processing =
+        innerJoin(processing, stateGroups, censusGeoidKey, tigerGeoidKey);
+    processing = processing.map((f) => combineWithSnap(f, snapKey, totalKey));
+    // Join with income.
+    // TODO: make income formatting prettier so it looks like a currency value.
+    //  Not trivial because it has some non-valid values like '-'.
+    processing = innerJoin(processing, incomePath, geoidTag, censusGeoidKey);
+    processing =
+        processing.map((f) => combineWithAsset(f, incomeTag, incomeKey));
+    // Join with SVI (data is at the tract level).
+    processing = processing.map(addTractInfo);
+
+    processing = innerJoin(processing, sviPath, tractTag, cdcGeoidKey);
+    processing = processing.map((f) => combineWithAsset(f, sviTag, sviKey));
+
+    // Get building count by block group.
+    const buildingsHisto =
+        computeBuildingsHisto(damageEnvelope, buildingPath, state, stateGroups);
+
+    // Create final feature collection.
+    processing = processing.map(
+        (f) => countDamageAndBuildings(f, damage, buildingsHisto));
+    allStatesProcessing = allStatesProcessing.merge(processing);
+  }
+
+  const assetName = 'data-ms-as-tot';
+  const scoreAssetPath = gdEePathPrefix + getDisaster() + '/' + assetName;
+  try {
+    ee.data.deleteAsset(scoreAssetPath);
+  } catch (err) {
+    if (err.message !== 'Asset not found.') {
+      throw err;
```

Do we want to throw an error when there isn't an old asset to delete? What about the first time running this script?

---

We're throwing an error only if the error is *not* "Asset not found". Rearranged to make clearer.